certbot-apache/certbot_apache/tests/configurator_test.py

```diff
@@ -517,12 +519,12 @@ def test_prepare_server_https_named_listen(self):
         # Test
         self.config.prepare_server_https("8080", temp=True)
         self.assertEqual(mock_add_dir.call_count, 3)
-        self.assertEqual(mock_add_dir.call_args_list[0][0][2],
-                         ["1.2.3.4:8080", "https"])
-        self.assertEqual(mock_add_dir.call_args_list[1][0][2],
-                         ["[::1]:8080", "https"])
-        self.assertEqual(mock_add_dir.call_args_list[2][0][2],
-                         ["1.1.1.1:8080", "https"])
+        call_args_list = [mock_add_dir.call_args_list[i][0][2] for i in range(3)]
+        self.assertEqual(
+            sorted(call_args_list),
+            sorted([["1.2.3.4:8080", "https"],
+                    ["[::1]:8080", "https"],
+                    ["1.1.1.1:8080", "https"]]))
```

Comment:
I'm not sure about this. Calls to add_dir_to_ifmodssl are randomized in Python 3 because ```prepare_server_https``` [uses a ```set```](https://github.com/certbot/certbot/blob/1fd847e/certbot-apache/certbot_apache/configurator.py#L674). Should the call order preserved? If so I'll gonna modify configurator.py instead.